packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -955,6 +926,16 @@ const VideoPlayerBase = class extends React.Component {
 		return true;
 	}
 
+	handleLoadStart = () => {
+		this.firstPlayReadFlag = true;
```

Comment:
Found a bug related to resetting internal values when a source has been updated.
In P&V app scenario, app loads the next video when it reaches the end of the video. So we need to reset `prevCommand`, and `speedIndex` so that ff rate does not show when the next video gets loaded when it reaches the end of the video.
Other internal values might need to be reset, but it didn't seeme necessary when I tested it.